### PR TITLE
don't mount the docker/crio sockets into build pods

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
/assign @adambkaplan 

@nalind fyi

**Comment by bparees** [comment]
@adambkaplan bump

/retest

File: pkg/build/controller/strategy/custom.go [hunk 1]
```diff
@@ -131,8 +131,8 @@ func (bs *CustomBuildStrategy) CreateBuildPod(build *buildv1.Build) (*corev1.Pod
 
 	if strategy.ExposeDockerSocket {
 		setupDockerSocket(pod)
```
Comment (adambkaplan) [inline pkg/build/controller/strategy/custom.go]: We're continuing to support docker-driven builds for the Custom strategy?

Comment (bparees) [reply pkg/build/controller/strategy/custom.go]: for now i'm intending to leave the mount in place.  If your host node has a /var/lib/docker.sock, then great, your custom builds keep working the way they didn't.  If it doesn't, they don't (and there's nothing we can do about it)

**Comment by bparees** [comment]
/retest


Ben Parees | OpenShift

**Pull request merged**